src/include/k5-spake.h

```diff
@@ -0,0 +1,107 @@
+/* -*- mode: c; c-basic-offset: 4; indent-tabs-mode: nil -*- */
+/* include/k5-spake.h - SPAKE preauth mech declarations */
+/*
+ * Copyright (C) 2015 by the Massachusetts Institute of Technology.
```

Comment:
Is this the right year?

Reply:
Looks like yes, based on the commit author date.  We've been at this for a while.